lab-aaron/src/component/category-form/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+
+class CategoryForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: props.category ? props.category.title : '',
+      id: props.category ? props.category.id : null,
+      timestamp: props.category ? props.category.timestamp : null,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    console.log('__FORM_PROPS__', this.props);
+  }
+
+  handleChange(e) {
+    this.setState({ title: e.target.value });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    this.props.onComplete(Object.assign({}, this.state));
+    this.setState({title: ''});
+  }
+
+  render() {
+    return (
+      <form className="category-form" onSubmit={this.handleSubmit}>
```

Comment:
something inside of your form is off, when I type in an expense name and then go to type in a price the price appears in both the name input box and the price input box.

Reply:
Yep, been trying to get some time to go figure it out and complete 27/28.
Been sick has been kicking my butt, but im planning on working through the weekend/weekends? until we are done to get them fixed.